Source/Scene/Cesium3DTileset.js

```diff
@@ -440,28 +393,29 @@ define([
             var childrenLength = children.length;
             var child;
             var k;
-            var additiveRefinment = (t.refine === Cesium3DTileRefine.ADD);
+            var additiveRefinement = (t.refine === Cesium3DTileRefine.ADD);
+
+            if (t.hasTilesetContent && t.isReady()) {
+                // If tile has tileset content, skip it and process its child instead (the tileset root)
+                child = t.children[0];
+                if (child.isContentUnloaded()) {
+                    if (outOfCore) {
+                        requestContent(tiles3D, child);
```

Comment:
Up to you, but it might be a tad cleaner to pass `outOfCore` to `requestContent` and test it there instead of checking it in three separate places like we do now.

In general, we should try to keep this tree traversal code as clean as possible since it is the most important function in 3D Tiles.